login.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <main class="login-main">
+    <div class="login-container">
+      <div class="login-image-container row">
+        <img src="assets/LogoWolox.png" alt="Logo Wolox" class="login-image-logo">
```

Comment:
Please, rename the `LogoWolox` asset by `logo-wolox`.

Reply:
Done